src/App.js

```diff
@@ -1,23 +1,18 @@
-import React, { Component } from "react";
+import React from "react";
 import { Provider } from "react-redux";
 import AppRouter from "./AppRouter";
 import store from "./store";
 import Notifier from "./components/Notifications/Notifier";
 
-// import './App.css';
+const App = () => (
+    <Provider store={store}>
+        <React.Fragment>
+            {/* <AppNavbar/> */}
```

Comment:
The idea was to have a Navbar in the future, but as the visual design planning phase was a bit sloppy, nothing is decided yet

Reply:
The Css Baseline suggestion is good, will add it in the future commit

Reply:
> The idea was to have a Navbar in the future, but as the visual design planning phase was a bit sloppy, nothing is decided yet

Then just remove it instead of commenting it. If it is added in the future, then it will be added there. Otherwise, this will just be code clutter.